exercises/pangram/canonical-data.json

```diff
@@ -49,6 +49,11 @@
          "description": "Panagram in alphabet other than ASCII",
          "input": "Широкая электрификация южных губерний даст мощный толчок подъёму сельского хозяйства.",
          "expected": false
+      },
+      {
+        "description": "test that count is case-insensitive",
```

Comment:
How about: "upper and lower case versions of the same character should not be counted separately"